pages.fr/common/go.md

```diff
@@ -0,0 +1,32 @@
+# go
+
+> Outil de gestion du code source du go.
```

Comment:
I propose this slight update (IMHO, it removes the ambiguity with [go, the game](https://en.wikipedia.org/wiki/Go_(game))):
```suggestion
> Outil de gestion du code source Go.
```